gspread/spreadsheet.py

```diff
@@ -19,37 +22,40 @@
 class Spreadsheet:
     """The class that represents a spreadsheet."""
 
-    def __init__(self, http_client: HTTPClient, properties: Dict[str, Any]):
+    def __init__(self, http_client: HTTPClient, properties: Dict[str, Union[str, Any]]):
         self.client = http_client
         self._properties = properties
 
         metadata = self.fetch_sheet_metadata()
+        import pprint
```

Comment:
imports should be at the top

is pprint a default library?

Reply:
oupsy :see_no_evil:   that was for debugging purpose